docs/source/API/containers/DualView.rst

```diff
@@ -207,6 +207,14 @@ Description
 
            dual_view_type::t_host hostView = DV.view<dual_view_type::t_host::memory_space>();
 
+    .. cppkokkos:function:: const t_host& view_host() const;
+
+       * Return the host-accessible View. With deprecated code enabled, return the View by value.
```

Comment:
```suggestion
       *  Return the host-accessible View. Returns the View by value with `Kokkos_ENABLE_DEPRECATED_CODE_4=ON`
```

Reply:
I agree I think it is worth mentioning the option `Kokkos_ENABLE_DEPRECATED_CODE_4`.